app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -95,6 +95,8 @@ class EventsFragment : Fragment() {
         showNoInternetScreen(isNetworkConnected())
 
         rootView.retry.setOnClickListener {
+            if (isNetworkConnected())
+                eventsViewModel.loadLocationEvents(eventsViewModel.savedLocation.toString())
```

Comment:
dont call `isNetworkConnected()`twice!

Reply:
showNoInternetScreen function is used to show the screen. I am just calling this once actually :+1:

Reply:
isNetworkConnected is being called in retryOnClickListener store it in some boolean and reuse

Reply:
I will prevent this :+1:

Reply:
Updated it